src/quo2/components/loaders/skeleton_list/view.cljs

```diff
@@ -0,0 +1,104 @@
+(ns quo2.components.loaders.skeleton-list.view
+  (:require [quo2.theme :as theme]
+            [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [react-native.masked-view :as masked-view]
+            [react-native.reanimated :as reanimated]
+            [quo2.components.loaders.skeleton-list.style :as style]
+            [reagent.core :as reagent]
+            [quo2.components.loaders.skeleton-list.constants :as constants]))
+
+(defn static-skeleton-view
+  [{:keys [index content color]}]
+  [rn/view
+   {:style (style/container content)}
+   [rn/view {:style (style/avatar color)}]
+   [rn/view {:style style/content-container}
+    [rn/view
+     {:style (style/content-view
+              {:type    (if (= content :list-items) :message :author)
+               :index   index
+               :content content
+               :color   color})}]
+    [rn/view
+     {:style (style/content-view
+              {:type    (if (= content :list-items) :author :message)
+               :index   index
+               :content content
+               :color   color})}]
+    (when (= content :notifications)
+      [rn/view
+       {:style (style/content-view {:type    :message2
+                                    :index   index
+                                    :content content
+                                    :color   color})}])]])
+
+(defn- f-animated-skeleton-view
+  [{:keys [style color skeleton-height animated? translate-x window-width] :as data}]
+  (let [loading-color (colors/theme-colors colors/neutral-10 colors/neutral-60)]
+
+    (rn/use-effect
+     (fn []
+       #(when-not animated?
+          (reanimated/cancel-animation translate-x))
+       (reanimated/animate-shared-value-with-repeat
```

Comment:
The args for this function call can be on the same line. I hope we'll one day revisit some auto-formatting aspects, especially the line limit of 120 chars, this causes more harm than good in tons of places.

```clojure
(rn/use-effect
 (fn []
   (reanimated/animate-shared-value-with-repeat translate-x window-width 1000 :linear -1 false)
   [animated?]))
```